### ISSUE TITLE
Add option to disable notifications

### ISSUE THREAD

**Issue by bfil** [AtomBuild/atom-build-cargo#86]
It might just be me, but I find the notifications shown by this package a little bit invasive and not very useful when activating the "build on save" flag, the "Building project" info notification is quite distracting in this case, when all I care about is the feedback from the linter to be displayed.

When doing TDD and using `test` as the build action if there are test failures it gets even worse, with panics and "test failed" error notifications popping out everywhere.

In most cases, from my workflow point of view the linter feedback and the green/red feedback at the bottom left of the editor are enough for me to know what's going on with "build on save" turned on, and if I need the details I can just decide to show the output panel on error for example.

For these reasons I would say it would be great to have the option to disable the notifications in the preferences. For now, I just hacked it in the source folder and removed the notifications code and everything looks great to me.

Would you be open to supporting this?

I know there are plans to make notifications less intrusive in Atom itself (https://github.com/atom/notifications/issues/29) but would still be great to have the option for now imo.

**Comment by oli-obk** [comment]
I don't think this is something we can do in build-cargo, since these messages are produced by the `build` package.

**Comment by bfil** [comment]
I'm referring to these: https://github.com/AtomBuild/atom-build-cargo/search?q=notifications.add

**Comment by oli-obk** [comment]
Ah... well the alternative is that these messages get swallowed without showing up anywhere (since these messages don't have a span for showing an inline lint)

The output panel is pretty much useless when using the json errors

**Comment by bfil** [comment]
Oh yeah well, I've disabled the JSON errors in the panel so I see the human readable output when I bring it up.

**Comment by Tectract** [comment]
I'm getting notifications about the file names of the file I am viewing when switching tabs. I would like an option to disable ALL NOTIFICATIONS in Atom. I hate notifications, they are extremely intrusive popups and do nothing but distract me and slow me down.

**Comment by achimbode** [comment]
For people who stumble upon this thread trying to get rid of the annoying popups - this package seems to do the job: **[notifications-plus](https://atom.io/packages/notifications-plus)**.
I think there is a difference between the _possibility_ of keeping track of errors of all kinds (sometimes thrown by extensions that I need and that apart from the errors work fine) and scattering the whole screen with it. This package allows for both at once. Thanks a lot to [UziTech](https://atom.io/users/UziTech) for sharing it!
